qa/build/scala.Dockerfile

```
@@ -47,7 +47,17 @@ RUN curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - && \
 # Install Node.js and npm
 RUN apt-get install -y nodejs npm && \
     npm install -g n && \
-    n 20
+    n 23
+
+# Upgrage yarn to latest version
+# https://yarnpkg.com/corepack
+RUN npm install -g corepack
```

Hadolint (2.12.0)

[warning] Pin versions in npm. Instead of `npm install <package>` use `npm install <package>@<version>`

(DL3016)